File: public/usage-examples/animations/free_animation_script_with_name-1-example.py
from splashkit import *

def main():
    # Register a named script (WalkingScript -> kermit.txt)
    load_animation_script_named('WalkingScript', 'Resources/animations/kermit.txt')

    # Free by name
    free_animation_script_with_name('WalkingScript')
    write_line('Animation script freed (by name).')

if __name__ == '__main__':
    main()

File: public/usage-examples/animations/free_animation_script_with_name-1-example.cpp
#include "splashkit.h"

int main()
{
    // Load the script under a named identifier
    load_animation_script_named("WalkingScript", "Resources/animations/kermit.txt");

    // Free the script by name when we're finished
    free_animation_script_with_name("WalkingScript");

    write_line("Animation script freed (by name).");
    return 0;
}
